src/eko/io/runcards.py

```diff
@@ -176,32 +208,49 @@ def targetgrid(self) -> interpolation.XGrid:
     def targetgrid(self, value: interpolation.XGrid):
         self._targetgrid = value
 
+    @classmethod
+    def from_dict(cls, dictionary: dict):
+        """Deserialize rotation.
+
+        Load from full state, but with public names.
+
+        """
+        d = dictionary.copy()
+        for f in fields(cls):
+            if f.name.startswith("_"):
+                d[f.name] = d.pop(f.name[1:])
+        return cls._from_dict(d)
+
+    @property
+    def raw(self):
+        """Serialize rotation.
+
+        Pass through interfaces, access internal values but with a public name.
+
+        """
+        d = self._raw()
+        for key in d.copy():
+            if key.startswith("_"):
+                d[key[1:]] = d.pop(key)
+
+        return d
+
 
 @dataclass
 class OperatorCard(DictLike):
     """Operator Card info."""
 
     mu0: float
     """Initial scale."""
+    mugrid: List[Target]
```

Comment:
```suggestion
    mugrid: List[Target]
    """Array of final scales."""
```

Reply:
This level of documentation is coming together with the type...

Reply:
Fine, but `mu` is not really telling, so let's just add that single sentence ...

Reply:
I'm not explicitly against that sentence, and for sure in favor of more telling names. If you have anything better than `mugrid` I'd be glad to adopt, even though I believe `mugrid` to be actually meaningful in this case.

My main point is that the content of your sentence is not much in the name, but rather in the type:
- *Array of* $\iff$ `List[]`
- *final scales* $\iff$ `Target`

Moreover, *Array* is wrong, since it is not an array, and *final scales* is wrong as well, since they are supposed to be pairs `(scale, nflavs)`, while the information in the type is much more accurate.

They are not especially wrong, they are only approximate. And since they are less accurate then type itself, nor adding more information, I don't see much the point.